docs/resources/mcs_db_user.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: "mcs"
+page_title: "mcs: db_user"
+sidebar_current: "docs-db-user"
+subcategory: ""
+description: |-
+  Manages a db user.
+---
+
+# mcs\_db\_user
+
+Provides a db user resource. This can be used to create, modify and delete db user.
+
+## Example Usage
+
+```terraform
+
+resource "mcs_db_user" "myuser" {
+  name        = "myuser"
+  instance_id = example_db_instance_id
+  password    = "password"
+  databases = [example_db_database_name, example_db_other_database_name]
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the user. Changing this creates a new user.
+
+* `instance_id` - (Required) ID of the instance that user is created for. Changing this creates a new user.
+
+* `password` - (Required) The password of the user.
```

Comment:
it's shame we propose users to have password in tf files. should we release this?

Reply:
can be resolved by terraform itself https://learn.hashicorp.com/tutorials/terraform/sensitive-variables